server/server.js

```diff
@@ -13,4 +13,12 @@ useMorganMiddleware(app);
 app.use(express.static(assets));
 app.use('/test', router);
 
+app.get('/*', (req, res) => {
+  res.sendFile(path.join(__dirname, '../client/'), (err) => {
```

Comment:
I believe __dirname gives you the root of your whole file system if I'm not mistaken, which brings up the question of using '../client/' syntactically. No qualms if intended behavior is produced.

Reply:
You still need to point to the next part of the path from the relative position of the file its being called in.

Reply:
The alternative is to use `path.resolve()`

Reply:
__dirname does give the root, and is the first argument of join.  `../client/` is supplied as the second argument.

My understanding is `join` concatenates all of the arguments, in order. The outcome is the server responds with the content in that concatenated filepath.